src/client/mod.rs

```diff
@@ -309,6 +310,27 @@ impl Client {
         pipeline: impl IntoIterator<Item = Document>,
         options: Option<ChangeStreamOptions>,
     ) -> Result<ChangeStream> {
-        todo!();
+        let pipeline: Vec<Document> = pipeline.into_iter().collect();
+        let mut options = options;
```

Comment:
Instead of rebinding `options` as mut, the parameter can just be defined as mut, i.e. `mut options: Option<ChangeStreamOptions>`

Reply:
fixed

Reply:
This still doesn't seem to be fixed in the latest commit.

Reply:
didn't push earlier. fixed now